xyz.anirban.ChatGPTElectron.yml

```diff
@@ -0,0 +1,63 @@
+app-id: xyz.anirban.ChatGPTElectron
```

Comment:
`anirban.xyz` looks like a parked domain. is it under the control of the developer of this project?

Reply:
ok i will use io.github.anirbandey1.ChatGPTElectron